src/pages/de/installation.md

```diff
@@ -16,17 +16,17 @@ Zu Demonstrationszwecken verwenden wir in den folgenden Beispielen [`npm`](https
 
 ## Astro-Projekt erzeugen
 
-`npm init astro` ist der einfachste Weg in einem neuen Projekt Astro zu installieren. Führe diesen Befehl in deinem Terminal aus, um unseren `create-astro`-Installationsassistenten zu starten, der dich bei der Einrichtung eines neuen Projekts unterstützt.
+`npm create astro@latest` ist der einfachste Weg in einem neuen Projekt Astro zu installieren. Führe diesen Befehl in deinem Terminal aus, um unseren `create-astro`-Installationsassistenten zu starten, der dich bei der Einrichtung eines neuen Projekts unterstützt.
 
 ```shell
-# Mit NPM
-npm init astro
+# Mit npm
+npm create astro@latest
 
-# Yarn
+# yarn
 yarn create astro
 
-# Pnpm
-pnpm create astro
```

Comment:
npm and pnpm prefer to be all lowercase. Changed yarn's casing for consistency as well